src/store/wizardSlice.ts

```diff
@@ -118,6 +125,18 @@ const initialState: wizardState = {
       enabled: [],
     },
   },
+  fileSystem: {
+    mode: 'automatic',
+    fileSystemConfiguration: [
+      {
+        id: uuidv4(),
+        mountpoint: '/',
+        fstype: 'xfs',
+        size: 10,
+        unit: UNIT_GIB,
+      },
+    ],
```

Comment:
This is great for initialization of manual partitioning, but we probably shouldn't use it for automatic. The question is can we / should we put it in a condition here? I mean we could probably put a condition to the requestMapper "if automatic, put only mode in the request". But that would mean that the store has this value in it for automatic partitioning, which could be confusing when debugging.

Reply:
good point, I think the good solution here is to initialize the configuration to empty array, and if we choose manual. we have the first row initizalization, WDYT @regexowl ?

Reply:
Sounds good to me :+1: